docs/library/network.WLAN.rst

```diff
@@ -133,4 +133,24 @@ Methods
    hostname       The hostname that will be sent to DHCP (STA interfaces) and mDNS (if supported, both STA and AP). (Deprecated, use :func:`network.hostname` instead)
    reconnects     Number of reconnect attempts to make (integer, 0=none, -1=unlimited)
    txpower        Maximum transmit power in dBm (integer or float)
+   pm             WiFi Power Management setting (see below for allowed values)
    =============  ===========
+
+Constants
+---------
+
+.. data:: WLAN.PM_PERFORMANCE
+        WLAN.PM_POWERSAVE
+        WLAN.PM_DEFAULT
+        WLAN.PM_NONE
+
+    Allowed values for the ``WLAN.config(pm=...)`` network interface parameter:
+
+        * ``PM_PERFORMANCE``: high performance wifi with higher power consumption
+        * ``PM_POWERSAVE``: reduce wifi power consumption with reduced
+          performance
+        * ``PM_NONE``: disable wifi power management
+        * ``PM_DEFAULT``: default power settings used by the wifi chip.
+
+    Some wifi chip drivers may also allow setting the *pm* config option to a
+    tuple of driver-specific parameters (for fine tuning power management).
```

Comment:
I suggest to remove the tuple capability for now.  We can add it later if needed.

Reply:
Done